snforge_std/src/cheatcodes.cairo

```diff
@@ -48,6 +48,10 @@ fn stop_prank(target: CheatTarget) {
     cheatcode::<'stop_prank'>(inputs.span());
 }
 
+fn replace_bytecode(contract: ContractAddress, new_class: ClassHash) {
+    cheatcode::<'replace_bytecode'>(array![contract.into(), new_class.into()].span());
+}
+
```

Comment:
We should add some `forge/integration` tests to make sure the cheatcode works on the Cairo side too. It can be very simple, but please add it